packages/at_secondary_server/lib/src/verb/handler/notify_verb_handler.dart

```diff
@@ -439,4 +446,8 @@ class NotifyVerbHandler extends AbstractVerbHandler {
     }
     return '${verbParam[FOR_AT_SIGN]}:${verbParam[AT_KEY]}${verbParam[AT_SIGN]}';
   }
+
+  bool _isAuthorizedToSendNotification(String? sharedBy, String currentAtSign) {
+    return sharedBy == currentAtSign;
```

Comment:
As a result of the code above, `verbParams[AT_SIGN] =
          AtUtils.formatAtSign(verbParams[AT_SIGN]) ?? currentAtSign;` this check will always succeed if the sharedBy atSign is not supplied in the verb params